util/perl/OpenSSL/Test.pm

```diff
@@ -831,6 +832,63 @@ sub openssl_versions {
     return @versions;
 }
 
+=over 4
+
+=item B<ok_nofips EXPR, TEST_NAME>
+
+C<ok_nofips> is equivalent to using C<ok> when the environment variable
+C<FIPS_MODE> is undefined, otherwise it is equivalent to C<not ok>. This can be
+used for C<ok> tests that should fail in C<FIPS_MODE> only. The parameters are
```

Comment:
Looking at the phrasing, "This can be used for XX tests that should fail in FIPS_MODE only" seems a bit curious.  They are certainly allowed to fail in non-FIPS mode as well!

May I suggest something like this? "This can be use for C\<ok> tests that must fail when testing a FIPS provider (see the B\<FIPS_MODE> environment variable in L\</ENVIRONMENT>, below)"

Reply:
The same goes for the documentation of `is_nofips` and `isnt_nofips` below

Reply:
I dont think it needs the 'see below' bit, but will change the first part.

Reply:
Cool.  Ping when you're done